services/integration/src/app.js

```diff
@@ -1,5 +1,5 @@
-const express = require('express');
-const mongoose = require('mongoose');
+const express = require("express");
```

Comment:
why were this changes applied?

Reply:
app.js I dont modify this file

Reply:
Git is saying that you did the change, please reverse those changes